tests/io/v3/vm/permissions/test_permissions.py

```diff
@@ -0,0 +1,60 @@
+'''
```

Comment:
Change filename to `test_permissions_api.py`
Reference: https://github.com/tenable/pyTenable/pull/502/files

Reply:
done